includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -291,6 +291,38 @@ public function isBlocked() {
 		$this->response->setData( [ 'blocked' => $user->isBlocked() ] );
 	}
 
+	public function logPiggybackAction() {
+		$this->response->setFormat( 'json' );
+		$this->response->setCacheValidity( \WikiaResponse::CACHE_DISABLED );
+		$this->response->setVal( self::FIELD_SUCCESS, false );
+
+		if ( !$this->authenticateViaTheSchwartz() ) {
+			return;
+		}
```

Comment:
More like a question than comment - we repeat the setFormat/setCacheValidity/authenticateViaTheSchwartz in each public method of this controller. Is there a way to set it for the whole controller in one place?